cfgov/v1/models/images.py

```diff
@@ -84,10 +84,19 @@ def resize(self, size):
         operation.run(mock_image, image=None, env={})
         return mock_image.width, mock_image.height
 
+    @property
```

Comment:
I believe it's necessary for the two places you commented above which don't use `ImageBasic`.